src/algebra/ring/equiv.lean

```diff
@@ -192,6 +192,9 @@ initialize_simps_projections ring_equiv (to_fun → apply, inv_fun → symm_appl
 
 @[simp] lemma symm_symm (e : R ≃+* S) : e.symm.symm = e := ext $ λ x, rfl
 
+@[simp]
+lemma coe_to_equiv_symm (e : R ≃+* S) : (e.symm : S ≃ R) = (e : R ≃ S).symm := rfl
```

Comment:
Are you sure this is the more useful direction? Do our other equiv types have a similar lemma we can be consistent with?

Reply:
This is why I asked you for review, I was hoping you would know! 
I tried to follow https://leanprover-community.github.io/mathlib_docs/algebra/algebra/basic.html#alg_equiv.to_ring_equiv_symm as I couldn't decide myself what would work best as a simp lemma.

Reply:
I guess consistency is a good enough criteria for now, if we can't identify a problem otherwise.